tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -187,8 +187,32 @@ def _run(self):
                                         hostname not in network_ip):
                                         b_index += 1
                                         continue
+                                    sub_vol_size = (int(volumes['volume%s.brickcount' % index])) / int(
+                                        volumes[
+                                            'volume%s.subvol_count' % index
+                                        ]
+                                    )
+                                    brick_name = NS.node_context.fqdn + ":" + volumes[
+                                        'volume%s.brick%s.path' % (
+                                            index, b_index
+                                        )
+                                    ].split(":")[-1].replace("/","_")
+
+                                    vol_brick_path = "clusters/%s/Volumes/%s/Bricks/subvolume%s/%s" % (
+                                        NS.tendrl_context.integration_id,
+                                        volumes['volume%s.id' % index],
+                                        str((b_index - 1) / sub_vol_size),
+                                        brick_name
+                                    )
+
+                                    NS._int.wclient.write(
+                                        vol_brick_path,
+                                        ""
```

Comment:
Since this is just a reference, just the directory path is sufficient. hence passing blank for value